sync/linux/controlplane.sh

```diff
@@ -154,6 +154,7 @@ cp $HOME/.kube/config /var/sync/shared/config
 rm -f /var/sync/shared/kubejoin.ps1
 
 cat << EOF > /var/sync/shared/kubejoin.ps1
+mv C:\Users\vagrant\crictl.exe "C:\Program Files\containerd"
```

Comment:
Can you skip if the file exists? 

We should handle crictl in the image

https://github.com/kubernetes-sigs/image-builder/issues/640